lib/rust/ensogl/core/src/display/world/stats.rs

```diff
@@ -41,30 +46,47 @@ impl {
             monitor.add( debug::monitor::SpriteSystemCount  :: new(&stats) ),
             monitor.add( debug::monitor::SpriteCount        :: new(&stats) ),
         ];
-        Self {stats,monitor,panels}
+        Self {stats,performance,forced_profiling,monitor,panels}
+    }
+
+    /// Force data measurements to be always performed, regardless of whether the monitor is
+    /// visible.
+    pub fn force_profiling(&mut self) {
+        self.forced_profiling = true;
     }
```

Comment:
Another thing we've been chatting about during last review. Do EnsoGL core know about the "profiling concept"? Answer is no, because profiling is built as a library on top of EnsoGL. So why these concepts are leaking here in your design?